src/style.css

```diff
@@ -1 +1,80 @@
 /* add styles here */
+@import 'fonts.css';
+
+body {
+  font-family: Avenirbook, sans-serif;
```

Comment:
The font-family name 'Avenirbook' should match the name defined in 'fonts.css', which is 'avenirbook' (case-sensitive). Update it to 'avenirbook' to ensure the font is applied correctly.